uber/model_checks.py

```diff
@@ -37,6 +37,30 @@ def has_email_address(account):
 
 
 @validation.Group
```

Comment:
Yeah, that was something I think I mentioned as a TODO item in the original PR that implemented ``@validation``.

The simplest approach (not necessarily the best) would be to define a second instance of the ``Validation`` class at
https://github.com/magfest/ubersystem/blob/master/uber/decorators.py#L316
like
```python
validation, prereg_validation = Validation(), Validation()
```

Then get rid of the ``validators`` property at
https://github.com/magfest/ubersystem/blob/master/uber/models.py#L232

which is only used in the ``check`` method anyway.  Instead of calling ``model.validators`` we could rewrite ``check``, which is at
https://github.com/magfest/ubersystem/blob/master/uber/utils.py#L63
to directly invoke the two different validations, taking an additional parameter to say whether or not it's a prereg validation.  It would look something vaguely like

```python
def check(model, *, prereg=False):
    # this first part is just exactly the same as what we have now
    for field, name in model.required:
        if not str(getattr(model, field)).strip():
            return name + ' is a required field'

    # this is basically just what we have now, but it checks both sets of validators if they say ``prereg=True``
    for v in [validation] + ([prereg_validation] if prereg else []):
        for validator in v[self.__class__.__name__].values()
            message = validator(model)
            if message:
                return message
```

If you did that, you could say ``@prereg_validation.Group`` and have it only run the validation if someone said ``check(group, prereg=True)``, which is presumably what you want.

I wanted to originally implement that, but time constraints, other excuses, etc.